receiver/awscontainerinsightreceiver/internal/stores/kubeletutil/kubeletclient.go

```diff
@@ -36,11 +37,22 @@ func NewKubeletClient(kubeIP string, port string, logger *zap.Logger) (*KubeletC
 	}
 	endpoint = endpoint + ":" + port
 
-	// use service account for authentication
-	clientConfig := &kubelet.ClientConfig{
-		APIConfig: k8sconfig.APIConfig{
-			AuthType: k8sconfig.AuthTypeServiceAccount,
-		},
+	clientConfig := &kubelet.ClientConfig{}
```

Comment:
it's a nit at this point, but it could be simplified:
```
clientConfig = &kubelet.ClientConfig{
	APIConfig: k8sconfig.APIConfig{
		AuthType:       k8sconfig.AuthTypeServiceAccount,
		KubeConfigPath: kubeConfigPath,
	},
}
if kubeConfigPath != "" {
    clientConfig.APIConfig.AuthType = k8sconfig.AuthTypeServiceAccount
}
```